.github/actions/setup-c8run/action.yml

```diff
@@ -96,8 +96,8 @@ runs:
 
     - uses: actions/setup-go@v5
       with:
-        go-version: '>=1.23.1'
-        cache: false  # disabling since not working anyways without a cache-dependency-path specified
+        go-version: ">=1.23.1"
```

Comment:
Related to https://github.com/camunda/camunda/issues/28266 

We should have a styling convention in place to avoid these formatting changes on PRs. @eamonnmoloney can you easily adjust your linter config to avoid these changes? 

Else considering the urgency I would be fine with fixing it later ;)